app/main.py

```diff
@@ -1 +1,44 @@
-# write your code here
+class SoftwareEngineer:
+    class_skills = []
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+        self.skills += self.class_skills
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    class_skills = ["JavaScript", "HTML", "CSS"]
```

Comment:
avoid hardcoding the skills in each classes. You should use attribute of instance. Please, follow checklist

Reply:
not fixed